rules/60-evdev.rules

```diff
@@ -8,7 +8,7 @@ IMPORT{builtin}="hwdb --subsystem=input --lookup-prefix=evdev:", \
   RUN{builtin}+="keyboard", GOTO="evdev_end"
 
 # AT keyboard matching by the machine's DMI data
-ENV{ID_INPUT_KEY}=="?*", DRIVERS=="atkbd", \
```

Comment:
I think "not yet assigned" and "now not assigned (at this point)" are the same thing. "now not" is not as easy to parse, but it's good enough.